packages/mds-types/index.ts

```diff
@@ -38,12 +38,213 @@ export const RULE_UNIT_MAP = {
+  /** Number of events registered within the bin, by type. */
+  event_counts: { [S in VEHICLE_EVENT]: number }
+  vehicle_counts: VehicleCountMetricObj
+  /** Number of trips in region, derived from distinct trip ids. */
+  trip_count: number
+  /** Number of vehicles with: {0 trips:count, 1 trip:count, 2 trips:count, ...] during bin. */
+  vehicle_trips_count: { [x: number]: number } | null
+  /** Number of events which out of compliance with time SLA. */
+  // TODO:  break into object with this binning, other event types not important. (?)
+  event_time_violations: LateMetricObj
+  /** Number of telemetry events out of compliance with distance SLA. */
+  telemetry_distance_violations: MetricCount
+  /** Number of event anomalies. */
+  // TODO:  break into object like so
+  bad_events: {
+    /** Number of invalid events (not matching event state machine). */
+    invalid_count: number | null
+    /** Number of duplicate events submitted. */
+    duplicate_count: number | null
+    /** Number of out-of-order events submitted (according to state machine). */
+    out_of_order_count: number | null
+  }
+  /** SLA values used in these calculations, as of start of bin. */
+  // TODO:  break into object like so:
+  sla: {
+    /** Maximum number of deployed vehicles for provider. Comes from Policy rules. */
+    // Typical SLA: 500-2000 vehicles
+    max_vehicle_cap: number
+    /** Minimum number of registered vehicles for provider. */
+    // Typical SLA: 100 vehicles
+    min_registered: number
+    /** Minumum number of trip_start events. */
+    // Typical SLA: 100 events???
+    // TODO: per day???
+    min_trip_start_count: number
+    /** Minumum number of trip_end events. */
+    // Typical SLA: 100 events???
+    // TODO: per day???
+    min_trip_end_count: number
+    /** Minumum number of telemetry events. */
+    // Typical SLA: 1000 events???
+    // TODO: per day???
+    min_telemetry_count: number
+    /** Maximum time between trip_start or trip_end event and submission to server. */
+    // Typical SLA: 30 seconds
+    // TODO: per day???
+    max_start_end_time: number
+    /** Maximum time between trip_enter or trip_leave event and submission to server. */
+    // Typical SLA: 30 seconds
+    max_enter_leave_time: number
+    /** Maximum time between telemetry event and submission to server. */
+    // Typical SLA: 1680 seconds
+    max_telemetry_time: number
+    /** Maximum distance between telemetry events when on-trip. */
+    // Typical SLA: 100 meters
+    max_telemetry_distance: number
+  }
+}
+
 export const PROPULSION_TYPES = Enum('human', 'electric', 'electric_assist', 'hybrid', 'combustion')
 export type PROPULSION_TYPE = keyof typeof PROPULSION_TYPES
 
 export const VEHICLE_STATUSES = Enum('available', 'reserved', 'unavailable', 'removed', 'inactive', 'trip', 'elsewhere')
 export type VEHICLE_STATUS = keyof typeof VEHICLE_STATUSES
 
+export const VEHICLE_STATUSES_ROW = ['available', 'reserved', 'unavailable', 'trip']
```

Comment:
I'd rather call this `RIGHT_OF_WAY_STATUSES`, I think in `mds-daily` we have that already defined, so would probably be worth it to update that to use this shared type as well!